micro_rpc_build/src/lib.rs

```diff
@@ -22,6 +22,36 @@ use anyhow::Context;
 use prost_build::{Method, Service};
 use std::path::Path;
 
+#[derive(Copy, Clone, Debug)]
+pub enum ReceiverType {
+    /// &mut self
+    RefMutSelf,
+
+    /// &self
+    RefSelf,
+}
+
+impl Default for ReceiverType {
+    fn default() -> Self {
+        ReceiverType::RefMutSelf
+    }
+}
```

Comment:
I think you can just annotate the default variant on the enum itself with `#[default]`. https://rust-lang.github.io/rfcs/3107-derive-default-enum.html